idunn/blocks/wikipedia.py

```diff
@@ -197,16 +197,20 @@ def init_wiki_es(cls):
     def get_wiki_info(cls, wikidata_id, lang, wiki_index):
         cls.init_wiki_es()
 
-        resp = cls._wiki_es.search(
-            index=wiki_index,
-            body={
-                "filter": {
-                    "term": {
-                        "wikibase_item": wikidata_id
+        try:
+            resp = cls._wiki_es.search(
+                index=wiki_index,
+                body={
+                    "filter": {
+                        "term": {
+                            "wikibase_item": wikidata_id
+                        }
                     }
                 }
-            }
-        ).get('hits', {}).get('hits', [])
+            ).get('hits', {}).get('hits', [])
+        except ConnectionError:
```

Comment:
I think we should also override Elasticsearch `timeout` and `max_retries` params. 10s and 3 retries is too much here.

Reply:
Good idea, done